histoqc/BaseImage.py

```diff
@@ -61,7 +76,10 @@ def __init__(self, fname, fname_outdir, params):
         self["dir"] = os.path.dirname(fname)
 
         self["os_handle"] = openslide.OpenSlide(fname)
-        self["image_base_size"] = self["os_handle"].dimensions
+        self["bounding_box"] = strtobool(params.get("bounding_box","False"))
```

Comment:
can you confirm that bounding_box is appearing as a column  in the output TSV file? will definitely want users to have this information in their output

Reply:
sorry, this confused me, i see now that i was referring to img_bbox appearing in the output file.

can we modify this variable from "bounding_box" to "enable_bounding_box" or something like that, to indicate its flag variable and differentiate it from img_bbox which will contain the actual bounding box?